tests/runner/origin/base_host.py

```
@@ -329,176 +342,159 @@ class RunHostAndProgramRes:
-			try:
-				process.kill()
-			except:
-				pass
+	fut_host = asyncio.ensure_future(wrap_host())
+	fut_proc = asyncio.ensure_future(wrap_proc())
+	await asyncio.wait([fut_host, fut_proc, asyncio.ensure_future(prob_died())])
 
+	exceptions: list[Exception] = []
+	try:
+		fut_host.result()
+	except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)